apps/typeform/lambda/app.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const path = require('path');
+const express = require('express');
+const fetch = require('node-fetch');
+const handleForms = require('./forms-handler');
+const handleWorkspaces = require('./workspaces-handler');
+const fetchAccessToken = require('./fetch-access-token');
+
+const deps = {
+  fetch
+};
+
+const app = express();
+
+const FRONTEND = path.dirname(require.resolve('typeform-frontend'));
+
+app.use('/forms', async (req, res) => {
+  const { authorization } = req.headers;
+  if (!authorization) {
+    res.status(401).send('Unauthorized');
```

Comment:
Optional: Doing this will automatically do what you have above.
```suggestion
    res.sendStatus(401);
```